CMakeLists.txt

```diff
@@ -19,11 +19,11 @@ endif()
 # AMReX 21.06+ supports CUDA_ARCHITECTURES with CMake 3.20+
 # CMake 3.18+: CMAKE_CUDA_ARCHITECTURES
 # https://cmake.org/cmake/help/latest/policy/CMP0104.html
-if(CMAKE_VERSION VERSION_LESS 3.20)
+#if(CMAKE_VERSION VERSION_LESS 3.20)
```

Comment:
This condition is still possible to hit; why comment it out?

Reply:
We want to use it unconditionally now, saw this first in ImpactX.